remo/base/static/base/css/app-fd4.less

```diff
@@ -477,40 +477,75 @@ label.required:after {
     animation-timing-function: linear;
 }
 
-#navigation-box ul {
-    float: right;
+#navigation-box {
     font-family: 'Open Sans Light';
-    overflow: hidden;
+    font-size: 15px;
 
     li {
-        float: left;
-        display: block;
-        position: relative;
-        padding: 0;
-        margin: 0;
-        border-right: 1px solid #d3d3d3;
+        list-style: none;
         line-height: 45px;
+        position: relative;
+    }
 
-        a {
-            font-size: 15px;
-            padding: 0 20px;
-            display: block;
+    a {
+        padding: 0 20px;
+        display: block;
+        &:focus {
+            outline: none;
+            text-decoration: underline;
+        }
+    }
 
-            &:focus {
-                outline: none;
-                text-decoration: underline;
+    >ul {
+        float: right;
+        margin-bottom: 0;
+        >li{
+            float: left;
+            border-right: 1px solid #d3d3d3;
+            &.last {
+                border-right:none;
+                a {
+                    padding: 0 0 0 20px;
+                }
             }
+        }    
+    }
+
+    .dropdown {
+        display: inline-block;
+    }
+
+    .dropbtn {
+        display: inline-block;
+    }
+
+    .dropdown-content {
+        display: none;
+        background-color: #eeeeee; 
+        min-width: 100%;
+        margin: 0;
+        position: absolute;
+        z-index: 99;
+        li {
+            float: none;
+        }
+        a {
+          color: black;
+          text-decoration: none;
         }
 
-        &.last {
-            border-right:none;
-            a {
-                padding: 0 0 0 20px;
-            }
+        a:hover, a:focus {
+            background-color: #fafafa;
         }
     }
+
+    .dropdown:hover .dropdown-content {
+        display: block;
+    }
 }
 
+
+
 @media only screen and (max-width: @breakMedium) {
```

Comment:
Could you try `@breakLarge` here? To avoid 'Labs' breaking on a second row when the `Dashboard` is visible. 

And make sure to adjust the `.last` element too, because now it has different padding than the rest items for these resolutions.